### DISCUSSION TITLE
The project agenda and todo items

### DISCUSSION THREAD

**Discussion by mmikhasenko** [ComPWA/polarimetry#3] [General]
(Not sure the discussion section is the best place for it)

 - [x] Implement the LHCb model
    - [x] Code the Bugg Breit-Wigners for Kpi S wave
    - [x] Set up the input format for the couplings
    - [x] Code the coupling conversion to the DPD convention
 - [x] Check implementation 
    - [x] Check decay rate fractions for the individual chains
    - [x] Check polarization asymmetries for the individual decay chains (just helicity couplings in the alpha expression)
 - [x] Averaged polarization asymmetry vector (an integral with the intensity)
    - [x] Compute it as a numerical convolution with the intensity
    - [x] Use the Lambda subsystem and the alignment chain, compute the average
    - [x] Use the Delta subsystem and compute

**Comment by redeboer** [comment]
Thanks for adding the data. I'll be writing https://github.com/ComPWA/compwa-org/pull/129 into a more general library form in this repo today (and the weekend ;)). That serves a bit like a playground for https://github.com/ComPWA/ampform/issues/277, but the design will remain simplified to describe the Λc decay. Then next week, we can work out the todo items you listed here.

**Comment by redeboer** [comment]
> Not sure the discussion section is the best place for it

I think these (private) discussions pages are perfect for just posting stuff quickly, just like in Slack etc. (The benefit as opposed to issues: support for comment threads.) They can then be converted into issues. And what I like about issues is that they serve as a ticket that can be kept track of, especially with the renewed [GitHub project boards](https://github.com/features/issues). For this project, I collect related issues in this [table board](https://github.com/users/redeboer/projects/2).

**Comment by redeboer** [comment]
> Code the Bugg Breit-Wigners for Kpi S wave

Just a thing I was wondering while I was decoding the JSON file: what is the precise formula for the Bugg and the Flatte1450? (Perhaps you already posted, but I don't have access to a good overview atm.)

I assume the Breit-Wigner is the same as in TR-018?

**Comment by mmikhasenko** [comment]
Yes, the Briet wigner is the same. The `BuggBreitWigner` is 
![image](https://user-images.githubusercontent.com/22725744/166654968-3fd68acb-26be-4df0-a864-06accf7bc796.png)
where
```
m(Kpi) = sqrt(sigma)
sA = mK^2-mpi^2/2
rho = 2q/m(Kpi)
q # is a breakup
```
Here is the [Julia implementation](https://github.com/redeboer/polarization-sensitivity/blob/ecad4e0c73843de4f5913f2c942b30c80aba29c1/julia/notebooks/model0.jl#L129)

The `Flatte1405` is also similar to BW, just with
```
Gamma = Gamma1+Gamma2
```
where the `Gamma1` and `Gamma2` are the S-wave energy-dep widths for `pK` and `piSigma`.
For the `pK` channel, q0 is computed using `piSigma` (senseless made-up stuff).
Here is the [Julia implementation](https://github.com/redeboer/polarization-sensitivity/blob/ecad4e0c73843de4f5913f2c942b30c80aba29c1/julia/notebooks/model0.jl#L153)

**Reply by mmikhasenko** [reply]
BuggBW is fixed in #13

**Comment by mmikhasenko** [comment]
The rates slowly approaching expected values.

That is the current situation
![image](https://user-images.githubusercontent.com/22725744/168589939-ea11e910-a35d-4853-9410-1ee1b4513226.png)

**Reply by redeboer** [reply]
Rates in the Julia code are not divided by I_tot, no? (Why not?)

**Reply by mmikhasenko** [reply]
(Sorry, I should read the question twice!)
yes, divided. They are just in %, i.e. multiplied by 100

**Reply by redeboer** [reply]
Ahh yes of course!

**Reply by redeboer** [reply]
Rate matrix on its way in #11, just trying to get the matplotlib visualization right :S

**Reply by mmikhasenko** [reply]
cool! I am trying to identify why you rates are different

**Comment by mmikhasenko** [comment]
I have done the transformations of the text that we agreed on at the last meeting (Did not read through yet 😆).

Some update on the remaining items:

- [x] some numbers + uncertainties, l.295 @redeboer 
- [x] figure 3: left + right @redeboer 
- [x] figure 2, + isobar frame (?) @mmikhasenko 
- [x] supplemental material `.json`: 18 models (I0, alpha_i) @redeboer 
- [x] a lot of polishing @mmikhasenko, @redeboer, Miriam
